tests/v1/spec_decode/test_k3_dspark_remote_speculator.py

```
@@ -0,0 +1,278 @@
+        seeds=seeds,
+        draft_step=draft_step,
+        draft_logits=draft_logits,
+    )
+
+    assert sampled.tolist() == [7, 8]
+    assert all(
+        torch.equal(actual, expected)
+        for actual, expected in zip(
+            recorded["args"],
+            (logits, idx_mapping, temperature, seeds, positions + 1000),
+        )
```

Ruff (0.16.3)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)